yaksh/file_utils.py

```diff
@@ -50,3 +50,30 @@ def extract_files(zip_file, path=None):
         zip_file.close()
         return zfiles, extract_path
 
+
+def is_csv(document):
+    try:
+        try:
+            content = document.read(1024).decode('utf-8')
+        except AttributeError:
+            document.seek(0)
+            content = document.read(1024)
+        sniffer = csv.Sniffer()
+        dialect = sniffer.sniff(content)
```

Comment:
Is this `dialect` variable used anywhere else? If not then it is unnecessary.

Reply:
Actually dialect can be used while reading csv but I am not using so will remove it.